webapp/portlet/src/main/webapp/skin/less/portlet/ActivityStream/Style.less

```diff
@@ -267,6 +267,21 @@
   }
 }
 
+.previewContainer {
+  height: 80vh;
+}
+
+.preview-attachment-action {
+  right: -7%;
+  top: -10%;
+  position: absolute;
+}
+.AttachmentCarouselPreview .v-image__image.v-image__image--cover {
+  background-size: contain!important;
+  background-color: white!important;
+  border-radius: 5px!important;
```

Comment:
all of thos can be made using attributes and re-usable classes I think

Reply:
done!